src/instructlab/eval/mt_bench_common.py

```diff
@@ -254,7 +254,7 @@ def chat_completion_openai(
         try:
             messages = conv.to_openai_api_messages()
             if (
-                merge_system_user_message
+                (merge_system_user_message or conv.name == "mistral")
```

Comment:
definitely solved for mistral (and mixtral). I wonder if there are other models that disallow system messages. If so, we could enum the names and extend that list. This patch is perfect as is though- just food for thought.